app/editor/src/features/admin/evening-overview/interfaces/IEveningOverviewSection.ts

```diff
@@ -0,0 +1,9 @@
+export interface IEveningOverviewSection {
+  id: number;
+  sourceId?: number;
+  seriesId?: number;
+  otherSource?: string;
```

Comment:
`otherSource` will never be undefined.

Reply:
Can't a section have a source and not an "other source"?

Reply:
or v.v?

Reply:
I see what you're saying in terms of matching the backend though, just logically put it as nullable there

Reply:
`otherSource` is a freeform fill value.  It's either an empty string or has a value.